Jenkinsfile

```diff
@@ -111,15 +111,15 @@ def buildHipClangJob(compiler, flags, image, prefixpath="/opt/rocm", cmd = ""){
             withDockerContainer(image: image, args: dockerOpts) {
                 timeout(time: 5, unit: 'MINUTES')
                 {
-                    sh 'PATH="/opt/rocm/opencl/bin/x86_64/:$PATH" clinfo'
+                    sh 'PATH="/opt/rocm/opencl/bin/:$PATH" clinfo'
```

Comment:
I am not sure if this will work in both rocm containers. Maybe we should just use the `clinfo` provided by ubuntu.

Reply:
We can just add both paths. I’ve had issues with the base clinfo, but that might just be me.